webapp/boards/.eslintrc.json

```diff
@@ -95,11 +562,84 @@
       2,
       3
     ],
-    "object-curly-spacing": [
+    "formatjs/no-multiple-whitespaces": 2,
+    "require-yield": 2,
+    "rest-spread-spacing": [
+      2,
+      "never"
+    ],
+    "semi": [
```

Comment:
I'm afraid of the diff and conflicts when we turn this on, but it looks like this rule might not actually be having an effect?

Reply:
`semi: "off"` is down in ts overrides, if/when that's removed, it'll be a grand ol' time! 

> `19738 errors`